src/pages/EditOrganizationPage.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from '@/fixtures/fixtures';
+import BasePageModel from '@/pages/BasePageModel';
+
+class EditOrganizationPage extends BasePageModel {
+  async isLoaded() {
+    await expect(this.page.getByText('Edit Organization')).toBeVisible();
+  }
+
+  get createOrganizationSuccessMessage() {
+    return this.page.locator('.message');
+  }
+
+  get deleteOrganizationButton() {
+    return this.page.getByRole('button', { name: 'Delete' });
+  }
+
+  async clickDeleteOrganization() {
+    await this.deleteOrganizationButton.click();
+    //this.page.on('dialog', (dialog) => dialog.accept());
```

Comment:
because it was failing on this line, and it was also failing when I put it before. As it works without it I believe that it is not needed in this case